search_that_hash/config_object.py

```diff
@@ -40,7 +40,7 @@ def api_config(hashes: str, sth_api: str = None):
 
 
 def default_config():
-    return {
+    config = {
```

Comment:
I'm confused. What is this for? All of these should be in the config file, right?

Reply:
Yeah good point...I'll add them all to it.